cypress/e2e/moderation.cy.ts

```diff
@@ -0,0 +1,101 @@
+import {
+  addComment,
+  closeModerationPlugin,
+  openModerationPlugin, openPopoverMenu,
+  reportAndVerifyRequest,
+  openAndSelectItemFromDropdown
+} from "./utils";
+
+const MANIFEST = `#EXTM3U
+#EXT-X-MEDIA:TYPE=AUDIO,GROUP-ID="audio",LANGUAGE="en",NAME="English",AUTOSELECT=YES,DEFAULT=YES,URI="${location.origin}/media/index_1.m3u8",SUBTITLES="subs"
+
+#EXT-X-STREAM-INF:PROGRAM-ID=1,BANDWIDTH=509496,RESOLUTION=480x272,AUDIO="audio",SUBTITLES="subs"
+${location.origin}/media/index.m3u8`;
+
+const moderateItems = [
+    {id: 1, label: 'Sexual Content'},
+    {id: 2, label: 'Violent Or Repulsive'},
+    {id: 3, label: 'Harmful Or Dangerous Act'},
+    {id: 4, label: 'Spam / Commercials'}
+];
+
+describe('Moderation plugin', () => {
+  beforeEach(() => {
+    // manifest
+    cy.intercept('GET', '**/a.m3u8*', MANIFEST);
+    // thumbnails
+    cy.intercept('GET', '**/width/164/vid_slices/100', {fixture: '100.jpeg'});
```

Comment:
can we use URL instead of file?

Reply:
@MosheMaorKaltura I checked in cypress documentation and here are the available objects for stubbing response- https://docs.cypress.io/api/commands/intercept#StaticResponse-objects . seems like we can use a string that contains the url instead of fixture.